tests/components/apple_tv/test_config_flow.py

```diff
@@ -1191,7 +1192,7 @@ async def test_reconfigure_update_credentials(hass: HomeAssistant) -> None:
 
     result = await hass.config_entries.flow.async_init(
         DOMAIN,
-        context={"source": "reauth"},
+        context={"source": "reauth", "entry_id": config_entry.entry_id},
```

Comment:
This block should be replaced by `result = await config_entry.start_reauth_flow(hass)`
(preferably in a preliminary PR)

Reply:
https://github.com/home-assistant/core/pull/129313